.github/workflows/enact-migration.yaml

```diff
@@ -38,13 +41,32 @@ jobs:
       INFURA_KEY: ${{ secrets.INFURA_KEY }}
       POLYGONSCAN_KEY: ${{ secrets.POLYGONSCAN_KEY }}
     steps:
+      - name: Get governance network
+        if: ${{ github.event.inputs.different_enact_network == 'true' }}
```

Comment:
Should we even check/have an input, or just use the mapping?

Reply:
Good point. You actually reminded me that we need this input to condition on the second Seacrest action. Basically, we should only show a second QR code if there is a second network to connect to. Will update.

Reply:
This change should be backwards-compatible in that if `inputs.different_enact_network == false`, this GH action will behave exactly like it did without this change.

Reply:
Done

Reply:
So 2 QR codes works then?

Reply:
Actually yeah, we could still get rid of the input here and simplify things.

Reply:
Ok got rid of the input. It will show the 2 QR codes only when it's on a network that has an auxiliary gov network (e.g. Polygon/Mumbai). Will test it manually again before merging.

Reply:
> So 2 QR codes works then?

Yeah 2 QR codes work. It can read from two different chains and write to whichever one you have up on your mobile wallet. I assume if you connect with two different mobile devices, you could write to both chains.

Reply:
Oh, yeah if you could do that I bet it would